### ISSUE TITLE
path to sextractor

### ISSUE THREAD

**Issue by andreasalamanos** [megalut/sewpy#19]
Hi, sorry for the simple question.
So, it returns me always the same error:
raise RuntimeError("Could not run SExtractor. Is the path '%s' correct ? If not, specify sexpath='/path/to/sextractor'" % self.sexpath)
RuntimeError: Could not run SExtractor. Is the path '..../Documents/sewpy/sewpy/sewpy.py' correct ? If not, specify sexpath='/path/to/sextractor'

is it what you mean with path/to/sextractor? the sewpy.py??
thanks in advance

**Comment by kuntzer** [comment]
Hi.
This is the path to the [SExtractor executable](https://www.astromatic.net/software/sextractor). 

Depending how you installed it, this could be ```sex```, ```sextractor``` or ```/usr/bin/sextractor```.

Does that help?

**Comment by andreasalamanos** [comment]
sorry, where do i find it (sex, sextractor)? its not located  in usr/bin/, i checked that.
 Is it the the link that you attached ("SExtractor executable")?

**Comment by kuntzer** [comment]
```sewpy``` is "only" a wrapper around source extractor (```SExtractor```). We haven't rewritten the whole thing in python. You have to first install it from the website I gave on your machine.

**Comment by astroknots** [comment]
I, too, am having a perpetual 

RuntimeError: Could not run SExtractor. Is the path '**path/to/sextractor**' correct ? If not, specify sexpath='/path/to/sextractor'

the thing is, I've put in for **path/to/sextractor** the path to the directory where I work in sextractor ('home/username/sextractor-2.19.5/', to be almost explicit). it still says it is unable to run SExtractor. I've put in alias sex="sextractor" in the .bashrc of both that directory and the directory where I am running the python notebook that uses sewpy, and I've added 'home/username/sextractor-2.19.5' to my $PATH and echod the path to make sure that it's in there. So, what EXACTLY is this looking for and how is it trying to find it?

**Comment by mtewes** [comment]
Hi, and sorry for the slow reaction.
From what you say you should add
sexpath = ''/home/username/sextractor-2.19.5/sextractor'
or something similar, i.e. the full path of the executable, including the name of the executable.
If the executable is called sextractor and you don't know where it is installed, try 'which sextractor' to find the full path.

**Comment by astroknots** [comment]
> Hi, and sorry for the slow reaction.
> From what you say you should add
> sexpath = ''/home/username/sextractor-2.19.5/sextractor'
> or something similar, i.e. the full path of the executable, including the name of the executable.
> If the executable is called sextractor and you don't know where it is installed, try 'which sextractor' to find the full path.

Yeah, I did that and no dice. HOWEVER, once I added the directory itself to my PATH, I was able to set sexpath="sextractor" and it finally ran. This is probably more of a problem with however my computer is set up (it's a departmental rather than a personal machine, so I have less total control). Hopefully, however, this may be a helpful note to future sewpy users.
